lib/vsc/administration/slurm/sync.py

```diff
@@ -47,9 +47,13 @@ def execute_commands(commands, allow_failure=False):
         logging.info("Running command: %s", command)
 
         # if one fails, we simply fail the script and should get notified
-        (ec, _) = RunNoShell.run(command)
+        (ec, output) = RunNoShell.run(command)
         if ec != 0 and not allow_failure:
-            raise SCommandException(f"Command failed: {command}")
+            # Check if "Nothing added" is in the output, if so, we can ignore the error
+            if output is not None and "Nothing added" in output:
```

Comment:
`if output` is enough instead of negative none checking

Reply:
I wanted to avoid checking `""`.